INSTALL.md

```diff
@@ -129,7 +129,7 @@ $ svn co https://svn.opentechstrategies.com/repos/ots/trunk/clients/lever-for-ch
 
 Then you need to get the OTS utilities that includes `get-bigdata`
 ```
-$ svn co https://svn.opentechstrategies.com/repos/ots/trunk/utils $OTS_DIR/
+$ svn co https://svn.opentechstrategies.com/repos/ots/trunk/utils $OTS_DIR/utils
```

Comment:
In a previous PR I mentioned using full words like "svn checkout" instead of "svn co".  I know I haven't always done so in the past myself, but Subversion commands are not widely known these days so it's probably good for us to avoid shorthand where possible.

Reply:
Oh, hmmm, also, perhaps I should move the `bigdata` script to `ots-tools`?  Yeah, I'll do that right now... Okay, done: [commit f4d6ae9b9b8e in ots-tools](https://github.com/OpenTechStrategies/ots-tools/commit/f4d6ae9b9b8ec4aeb1bd20d4740b63fd62392ac6) and r17138 in Subversion.

Please adjust accordingly :-).

Reply:
(Note that r17139 in SVN adds a redirect script, though that is irrelevant to this PR.)

Reply:
This is what I get for fixing unrelated typos in my PRs ;)

Reply:
Updated in both locations to point to github instead!